? newk_GD.patch
Index: Makefile.PL
===================================================================
RCS file: /home/cvs/alternate_sgf/Makefile.PL,v
retrieving revision 1.7
diff -u -r1.7 Makefile.PL
--- Makefile.PL	25 Mar 2004 17:30:20 -0000	1.7
+++ Makefile.PL	25 Mar 2004 17:33:51 -0000
@@ -27,6 +27,7 @@
 
         # I didn't want to have to do this...
         # apparently use ::GD was getting executed before my eval()s ... 
-        'GD' => 2.12,
+
+        # 'GD' => 2.12,
     },
 );
Index: SGF2misc.pm
===================================================================
RCS file: /home/cvs/alternate_sgf/SGF2misc.pm,v
retrieving revision 1.70
diff -u -r1.70 SGF2misc.pm
--- SGF2misc.pm	25 Mar 2004 17:30:20 -0000	1.70
+++ SGF2misc.pm	25 Mar 2004 17:33:51 -0000
@@ -457,6 +457,7 @@
     return "$b</table>";
 }
 # }}}
+=cut
 # as_image {{{
 sub as_image {
     my $this = shift;
@@ -508,6 +509,7 @@
     $image->save($opts{filename});
 }
 # }}}
+=cut
 # as_freezerbag {{{
 sub as_freezerbag {
     my $this = shift;
